src/export_from_inboveg/query_inboveg.Rmd

```diff
@@ -35,42 +35,20 @@ update <- TRUE
  
 connection_inboveg <- connect_inbo_dbase(database_name = "D0010_00_Cydonia")
 
-classif_mhq <- bind_rows(
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Grasland", collect = TRUE),
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Moerassen", collect = TRUE),
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Grasland_BHM", collect = TRUE),
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Moeras_BHM", collect = TRUE),
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Duinen_BHM", collect = TRUE),
-    get_inboveg_classification(connection = connection_inboveg, survey_name = "N2000meetnet_Bos_91E0_sf", collect = TRUE)
-) %>%
+mhq_terr_surveys <- c("N2000meetnet_Grasland", "N2000meetnet_Moerassen", "N2000meetnet_Grasland_BHM", "N2000meetnet_Moeras_BHM", "N2000meetnet_Duinen_BHM", "N2000_Duinen_Fieldapp", "N2000meetnet_Bos_91E0_sf", "N2000meetnet_Heide_BHM", "N2000meetnet_Heide_BHM_2022", "N2000_grasland_fieldapp_18", "N2000_grasland_fieldapp_3")
```

Comment:
@Patrikoosterlynck: ik heb de lijst met inboveg surveys die opnames bevatten voor meetnet habitatkwaliteit geupdate.

Zijn de surveys  N2000_grasland_fieldapp_18 en N2000_grasland_fieldapp_3 ook relevant?

Reply:
Toon, N2000_grasland_fieldapp_18 niet en N2000_grasland_fieldapp_3 zijn niet van belang.
In je lijstje ontbreekt wel nog N2000_Duinen_fieldapp maar deze heeft een andere survey structuur (alle data zit in een classic opname) 
Verder zit ook nog data voor de heides in N2000_Heide_BHM en N2000_Heide_BHM_2022 en voor 6510 in N2000_6510_droogte die wij deden ipv ANB

Reply:
OK. N2000_Duinen_fieldapp staat ook in het lijstje. De output van de queries heeft hier op eerste zicht dezelfde structuur als de a andere surveys.